lightning/src/ln/chan_utils.rs

```diff
@@ -80,6 +80,50 @@ pub fn build_commitment_secret(commitment_seed: &[u8; 32], idx: u64) -> [u8; 32]
 	res
 }
 
+/// Build a closing transaction
+pub fn build_closing_transaction(value_to_holder: u64, value_to_counterparty: u64, holder_shutdown_script: Script, counterparty_shutdown_script: Script, funding_outpoint: OutPoint) -> Transaction {
+	let txins = {
+		let mut ins: Vec<TxIn> = Vec::new();
+		ins.push(TxIn {
+			previous_output: funding_outpoint,
+			script_sig: Script::new(),
+			sequence: 0xffffffff,
+			witness: Vec::new(),
+		});
+		ins
+	};
+
+	let mut txouts: Vec<(TxOut, ())> = Vec::new();
+
+	if value_to_counterparty > 0 {
+		txouts.push((TxOut {
+			script_pubkey: counterparty_shutdown_script,
```

Comment:
(here and in channel.rs) Calling this the "shutdown" script is a misnomer - the payout script is different from the shutdown one.

Reply:
Done

Reply:
I must've been high, I'm not sure what this means. Either script or shutdown_script are fine...